CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
```

Comment:
How should it look like? Do we separate it somehow to indicate what was done for which component, or is everything just put with each other?

Reply:
It depends on how we want our release notes to look like. I'm fine with including separate subheadins for forge and cast if we want to keep that separation.

Reply:
I think we should separate those, it will be cleaner